packages/garbo/src/diet.ts

```diff
@@ -184,9 +219,18 @@ function drinkSafe(qty: number, item: Item) {
       );
     }
   }
-  consumeWhileRespectingMoonRestaurant(() => {
-    if (!drink(qty, item)) throw "Failed to drink safely";
-  }, item);
+  if (item.notes?.includes("BEER") && canBusk()) {
+    for (let i = 0; i < qty; i++) {
+      buskForSaltyMouth();
+      consumeWhileRespectingMoonRestaurant(() => {
+        if (!drink(1, item)) throw "Failed to drink safely";
+      }, item);
+    }
+  } else {
+    consumeWhileRespectingMoonRestaurant(() => {
+      if (!drink(qty, item)) throw "Failed to drink safely";
+    }, item);
+  }
```

Comment:
```suggestion
  consumeWhileRespectingMoonRestaurant(() => {  
    if (item.notes?.includes("BEER") && canBusk()) {
      for (let i = 0; i < qty; i++) {
        buskForSaltyMouth();
        if (!drink(1, item)) throw "Failed to drink safely";
      }
    } else if (!drink(qty, item)) throw "Failed to drink safely";
  }, item);
```
this will need linting, probably